src/test/java/org/openmrs/contrib/qaframework/automation/FindPatientSteps.java

```diff
@@ -40,9 +40,9 @@ public void noPatients() {
 		assertNotNull(getElement(By.className("dataTables_empty")));
 	}
 
-	@And("User enters John Smith")
-	public void enterJohnSmith() {
-		findPatientPage.enterPatient("John Smith");
+	@And("User enters {string} patientName")
+	public void enterPatientName(String patientName) {
```

Comment:
@sherrif10 why not renaming the method with a genericName that can cater for searching a patient using patientIdentifier or patientName ?